Tests/Unit/Service/DumperTest.php

```diff
@@ -0,0 +1,265 @@
+<?php
+
+namespace Presta\SitemapBundle\Tests\Unit\Service;
+
+use PHPUnit\Framework\TestCase;
+use Presta\SitemapBundle\Event\SitemapPopulateEvent;
+use Presta\SitemapBundle\Service\Dumper;
+use Presta\SitemapBundle\Sitemap\Url\UrlConcrete;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Symfony\Component\Filesystem\Filesystem;
+
+class DumperTest extends TestCase
+{
+    private const DUMP_DIR = __DIR__ . '/.artifacts';
+
+    /**
+     * @var EventDispatcher
+     */
+    private $eventDispatcher;
+
+    /**
+     * @var Filesystem
+     */
+    private $filesystem;
+
+    /**
+     * @var Dumper
+     */
+    private $dumper;
+
+    private static function createDir(): void
```

Comment:
Shouldn't you move it after the `public` methods to comply with standards?

Reply:
Not sure why I had this in my mind, but static was a whole section.
I was not able to find anything on this particular point, and after rereading the code this sounds weird.
Will fix